src/hypertrace/agent/init/__init__.py

```diff
@@ -290,23 +287,12 @@ def set_zipkin_processor(self) -> None:
 
             logger.info('Added ZipkinExporter span exporter')
         except Exception as err:  # pylint: disable=W0703
-            logger.error('Failed to register_processor: exception=%s, stacktrace=%s',
+            logger.error('Failed to register exporter: exception=%s, stacktrace=%s',
                          err,
                          traceback.format_exc())
 
-    def set_otlp_processor(self) -> None:
+    def init_otlp_exporter(self) -> None:
         '''configure otlp span exporter + processor'''
-        if 'HT_TRACES_EXPORTER' in os.environ:
```

Comment:
This kind of logic is already took by config https://github.com/hypertrace/pythonagent/blob/main/src/hypertrace/agent/config/__init__.py#L114

Reply:
This still needs to be check to see if each of these exporters is configured. 

Slight miscalculation, but we updated this in another branch. See https://github.com/hypertrace/pythonagent/pull/189.

We can close this PR.